package.json

```diff
@@ -22,7 +22,7 @@
     "build": "tsc",
     "pretest": "npm run build",
     "prepublish": "npm run build",
-    "test": "mocha --opts test/mocha.opts dist/test",
+    "test": "mocha --opts test/mocha.opts dist/test/*.js && mocha --opts test/mocha.opts dist/test",
```

Comment:
Yes. There was no recursive option set in `test/mocha.opts` and I am not so keen to Mocha. I'll remove this.